### PR TITLE
Updating version after Chef 12.9 release

### PR DESCRIPTION
**Pull request by tyler-ball**
\cc @danielsdeleo @randomcamel @chefsalim @mwrock

### CODE REVIEW

File: omnibus/config/projects/chef.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 # Chef Release version pinning
 override :chef, version: "local_source"
-override :ohai, version: "v8.14.0"
+override :ohai, version: "master"
```
Comment (danielsdeleo) [inline omnibus/config/projects/chef.rb]: do we actually want to ride master here?

**Comment by danielsdeleo** [comment]
:+1: thought we wanted to keep everything, including ohai, on some kind of current or release version, but we don't really have a current for ohai. I'm happy either way, just wondering what the plan is.

File: omnibus/config/projects/chef.rb (hunk 1 above)
Comment (randomcamel) [inline omnibus/config/projects/chef.rb]: What he said.

**Comment by randomcamel** [comment]
👍

**Comment by tyler-ball** [comment]
Once @jkeiser gets his one-gemfile-to-rule-them-all changes merged in, it will float on the latest released version.  But for now it is still set on master

**Pull request merged**